targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF51822_UNIFIED/sdk/softdevice/s130/headers/ble.h

```diff
@@ -0,0 +1 @@
+#include "nrf_ble.h"
```

Comment:
Did you noticed that softdevice headers names were changed from 2.0.0 to 5.0.0? e.g. nrf_ble.h -> ble.h. and so one.
So you prefer to have code with #if-#else than file redirecting headers - right?